Templates/BlobTrigger-Batch/metadata.json

```diff
@@ -5,7 +5,6 @@
     "language": "Batch",
     "category": [ "Experimental" ],
     "userPrompt": [
-        "connection",
-        "path"
```

Comment:
That was actually a mistake - good catch :) I had removed 'path' from the other templates since not needed there. Will add back.